### PR TITLE
preserve volume name on pvc update

### PR DESCRIPTION
**Pull request by cristian-radu**
- **What does the PR do?** *(JIRA reference, background context, references to other tickets)*
- **What automated testing exists for this change?** *Link to new or existing unit tests*
- **Is there documentation?** *If yes, please provide a link to the docs.prsn.io page or appropriate Readme*
- **Any dependencies?** *(PRs in other repos that need to be merged alongside this)*
- [x] **Has the changelog been updated?**

### CODE REVIEW

**Comment by cristian-radu** [comment]
kube populates pvc.Spec.VolumeName after the volume has been created. we need to preserve this value when calling pvc.Update() because it is immutable.

**Comment by bsomogyi** [comment]
LGTM

File: pkg/util/k8s/persistent_volume_claim.go [hunk 1]
```diff
@@ -44,6 +44,7 @@ func (client *PersistentVolumeClaim) Update(resource *v1.PersistentVolumeClaim)
 		return err
 	}
 	resource.ResourceVersion = current.GetResourceVersion()
+	resource.Spec.VolumeName = current.Spec.VolumeName
```
Comment (geriBatai) [inline pkg/util/k8s/persistent_volume_claim.go]: Can we log.warning here?

Comment (cristian-radu) [reply pkg/util/k8s/persistent_volume_claim.go]: I added some more user friendly logging

**Comment by geriBatai** [comment]
LGTM

**Comment by geriBatai** [comment]
Aaand LGTM again

**Pull request merged**